src/deprecated/withStyles.jsx

```diff
@@ -0,0 +1,218 @@
+import React from 'react';
```

Comment:
Is this file completely untouched? I.e., you just moved what was previous at `src/withStyles.jsx` into this deprecated folder? Or did you change other things around as well?